packages/mobile/src/in-house-liquidity/index.ts

```diff
@@ -36,7 +40,13 @@ const getSerializedSignature = async (
   const signature = signMessage(message, privateKey, accountMTWAddress)
   return serializeSignature(signature)
 }
-const getAuthHeader = async (message: string, accountMTWAddress: string): Promise<string> => {
+const getAuthHeader = async (
+  endpoint: string,
+  accountMTWAddress: string,
+  body: string = ''
+): Promise<string> => {
+  const dateHeader = new Date().getUTCDate()
```

Comment:
This won't work because IHL will expect the date header of the request to match the date that is used in the signed message. It's mean to help prevent replay attacks.

Reply:
Ah, I understand how.